src/borg/archiver.py

```diff
@@ -4096,6 +4096,10 @@ def define_borg_mount(parser):
         have the exact same semantics as in "borg create". If PATHs are specified the
         resulting archive will only contain files from these PATHs.
 
+        ``--exclude-if-present`` just checks for files that are present in the archive,
+        it will not recognize new tag files in the local file system that have been
+        added after your archives were created.
```

Comment:
true, but isn't that a bit too specific?

recreate always only looks at the stuff in the input archive and never into the filesystem.
so **all** options and arguments refer to the input archive and not to the filesystem.